src/components/Slider/index.js

```diff
@@ -158,14 +158,16 @@ const Slider = ({ children, ...props }) => {
   )
 }
 
-Slider.Item = ({ as, ...props }) => (
+export const Item = ({ as, ...props }) => (
```

Comment:
const Item = ....

Slide.Item = Item ?

Reply:
@DorianMaliszewski isn't it what I'm already doing ?

```js
export const Item = ({ as, ...props }) => (
  <StyledBorderWrapper as={as} {...props} draggable="true" />
)

Slider.Item = Item
```